.circleci/config.yml

```diff
@@ -12,9 +12,12 @@ jobs:
      steps:
        - checkout
        - run: ci/do_circle_ci.sh bazel.release
+       - run: ci/do_circle_ci.sh github_release
```

Comment:
I think this will be run on every repository event, resulting in a release created for every merge?

We've historically handled this by adding a check in the script to only publish/release for tag events. [Here's an example](https://github.com/envoyproxy/envoy/blob/master/ci/docker_tag.sh#L7)

Reply:
this should be fixed, great catch!